research/gaia/gremlin/gs_gremlin/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "gs_gremlin"
+version = "0.1.0"
+edition = "2018"
```

Comment:
any consideration to define the edition as 2018?

Reply:
This is standard Rust at this stage. This version is recommended by Rust official.